9.mAP/notebooks/1.aggregate_map_scores_class_level.ipynb

```diff
@@ -0,0 +1,701 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # aggregate single cells scores with cell UUID
</pre><p>Update this <code>aggregating single cells using Metadata Well</code></p><p>In order to improve clarity </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/112/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>